policyengine_us/parameters/gov/states/nm/tax/income/credits/ctc/amount.yaml

```
@@ -20,28 +24,55 @@ brackets:
       # If the AGI is less than 0, the filer may claim the full credit
       2023-01-01: -.inf
     amount:
-      2023-01-01: 600
+      values:
+        2023-01-01: 600
+        2024-01-01: 622
+      metadata:
+        uprating: gov.irs.uprating
   - threshold:
       2023-01-01: 25_001
```

And elsewhere - instead let's use `p.calc(..., right=True)` in the formula. This more closely follows the law.
```suggestion
      2023-01-01: 25_000
```